Update: Added dynamic subdomain replace in host URL

anemeswmf opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

anemeswmf committed:
Add AlisQI certified-connector

anemeswmf committed:
Update: readme file

anemeswmf committed:
Merge remote-tracking branch 'upstream/dev' into dev
merge
git add .

anemeswmf committed:
Merged upstream/dev into dev and added necessary changes

vmanoharas commented:
Hello @anemeswmf, 

Can you please correct this folder structure by removing the folder .gitignore to proceed the review further?

![image](https://github.com/microsoft/PowerPlatformConnectors/assets/137378994/b236d86a-4122-4a83-8a49-c924ed36baa3)

anemeswmf committed:
Removed .gitignore file as requested by reviewer

anemeswmf commented:
Hello,

Removed .gitignore file from the folder structure and added a new commit: c77117e1ebc65998b8a3ca1f86916a0731b3c64f.

Submitted update in Connector Certification Portal.

Thank you,

-- Alexandru Nemes


From: vmanoharas ***@***.***>
Date: Thursday, 20 June 2024 at 12:50
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: Alexandru Nemes ***@***.***>, Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] Update: Added dynamic subdomain replace in host URL (PR #3543)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Hello @anemeswmf<https://github.com/anemeswmf>,

Can you please correct this folder structure by removing the folder .gitignore to proceed the review further?

image.png (view on web)<https://github.com/microsoft/PowerPlatformConnectors/assets/137378994/b236d86a-4122-4a83-8a49-c924ed36baa3>

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/PowerPlatformConnectors/pull/3543#issuecomment-2180272256>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2W6LAGTV5STCB63SBOWPZLZIKQWFAVCNFSM6AAAAABJSHUN2SVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCOBQGI3TEMRVGY>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

makeitfuture SRL

PTA. MIHAI VITEAZU, NR.3-4, SUBSOL,
AP.1, MUN. CLUJ-NAPOCA, CLUJ
jud Cluj
Represent Tiberiu Socaci
VAT ID RO45402628
Register No.: J12/6204/2021

***@***.***



This e-mail contains confidential and/or proprietary information. We would like to point out that your data will be processed within the scope of this e-mail; our data protection declaration applies. Your data is necessary for the processing of your request and for the usual business transaction. Further information can be requested at contact at makeitfuture.com. If you are not the correct addressee, or have received this email in error, please notify the sender and delete this email. Unauthorized copying and unauthorized disclosure of this e-mail and the information contained therein is not permitted.

vmanoharas commented:
Hello @anemeswmf, 

Thank you very much for your quick response!

your commit is not in a proper folder structure, kindly make this at root folder by removing other folders. 

![image](https://github.com/microsoft/PowerPlatformConnectors/assets/137378994/e968641d-0e03-4047-ba8d-76f9d7133403)

anemeswmf commented:
Hi @vmanoharascan you kindly guide me what commands to run or what exact steps to perform to achieve that? Thank you

anemeswmf commented:
Hi @vmanoharas could you kindly guide me what commands to run or what exact steps to perform to achieve that? Thank you

anemeswmf commented:
Hello,

Could you kindly guide me what commands to run or what exact steps to perform to achieve that?

Thank you,

-- Alexandru Nemes


From: vmanoharas ***@***.***>
Date: Thursday, 20 June 2024 at 18:55
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: Alexandru Nemes ***@***.***>, Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] Update: Added dynamic subdomain replace in host URL (PR #3543)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Hello @anemeswmf<https://github.com/anemeswmf>,

Thank you very much for your quick response!

your commit is not in a proper folder structure, kindly make this at root folder by removing other folders.

image.png (view on web)<https://github.com/microsoft/PowerPlatformConnectors/assets/137378994/e968641d-0e03-4047-ba8d-76f9d7133403>

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/PowerPlatformConnectors/pull/3543#issuecomment-2181035808>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2W6LADOZNZJZKYG7VNLUFDZIL3NXAVCNFSM6AAAAABJSHUN2SVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCOBRGAZTKOBQHA>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

makeitfuture SRL

PTA. MIHAI VITEAZU, NR.3-4, SUBSOL,
AP.1, MUN. CLUJ-NAPOCA, CLUJ
jud Cluj
Represent Tiberiu Socaci
VAT ID RO45402628
Register No.: J12/6204/2021

***@***.***



This e-mail contains confidential and/or proprietary information. We would like to point out that your data will be processed within the scope of this e-mail; our data protection declaration applies. Your data is necessary for the processing of your request and for the usual business transaction. Further information can be requested at contact at makeitfuture.com. If you are not the correct addressee, or have received this email in error, please notify the sender and delete this email. Unauthorized copying and unauthorized disclosure of this e-mail and the information contained therein is not permitted.

vmanoharas commented:
Hello @anemeswmf, 

Kindly follow the readme from the below URL, it has step by step guide. 

https://github.com/microsoft/PowerPlatformConnectors

anemeswmf committed:
Removed .DS_Store files

anemeswmf commented:
Hi @vmanoharas made the changes. Thank you

vmanoharas reviewed (approved):
Dear Partner, 

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- [Announcing Partner Center to certify and publish Power Platform Copilot connectors and plugins](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fblog%2Fannouncing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins%2F&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390444301%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=5MYg0GlCsic2JqzwJU0h7mIOc7uHOeL7ZWSbgq%2FFkWk%3D&reserved=0)

As next steps: 

1.	Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
2.	Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
3.	Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
4.	Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0)
5.	Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
6.	If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)

For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)

Many thanks,
Power Platform 
Copilot connector & plugin certification team

anemeswmf commented:
Hello,

We have noticed that the certification process for Power Platform connectors has changed from the previous ISV Studio-based approach. We were at the step for testing the connector in the preview region in the ISV Studio process.
Additionally, we recently updated the files for our connector, submitted a new pull request, which has now been approved and merged.
To avoid any rework and ensure a smooth transition to the new process, could you please clarify at what step we would be now in the new certification process?
For reference, this was our position in the ISV Studio process:

  *   Our initial pull request was approved and merged.
  *   We submitted the files to ISV Studio using our latest commit ID.
  *   We were notified to create an environment in the preview region to test our connector after functional verification by Microsoft.
  *   We submitted a new pull request for updates, which has now been approved and merged.
Here is the new process we have noted:

  1.  Learn about the new certification experience
  2.  Package your connector and plugin files
  3.  Initiate a certification request in Partner Center
  4.  Ensure compliance with Marketplace policies
  5.  Test your connector post certification for go-live deployment
  6.  Update your connector or plugin in the future
Could you please guide us on how to proceed in the new process without reworking the connector?
Thank you very much for your assistance.
Best regards,
-- Alexandru Nemes


From: vmanoharas ***@***.***>
Date: Tuesday, 25 June 2024 at 19:30
To: microsoft/PowerPlatformConnectors ***@***.***>
Cc: Alexandru Nemes ***@***.***>, Mention ***@***.***>
Subject: Re: [microsoft/PowerPlatformConnectors] Update: Added dynamic subdomain replace in host URL (PR #3543)

@vmanoharas approved this pull request.

Dear Partner,

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- Announcing Partner Center to certify and publish Power Platform Copilot connectors and plugins<https://powerautomate.microsoft.com/en-us/blog/announcing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins/>

As next steps:

  1.  Learn about the new certification experience- Get your Power Platform connector and plugin certified - Overview<https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-certification>
  2.  Package your connector and plugin files- Prepare Power Platform connector and plugin files for certification<https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission>
  3.  Initiate a certification request in Partner Center- Verified publisher certification process<https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification>
  4.  Ensure your connector & plugin files comply to the Marketplace policies here- 5000 Power Platform Connector Policies for Marketplace<https://learn.microsoft.com/en-us/legal/marketplace/certification-policies#5000-power-platform-connectors>
  5.  Test your connector post certification to provide go-live for deployment- Testing Guidelines<https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-testing>
  6.  If you wish to Update your connector or plugin in future- Updating Guidelines<https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-updates>

For any queries or concerns please get in touch with us on ***@***.******@***.***>

Many thanks,
Power Platform
Copilot connector & plugin certification team

—
Reply to this email directly, view it on GitHub<https://github.com/microsoft/PowerPlatformConnectors/pull/3543#pullrequestreview-2139211872>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A2W6LAFAKTURLZYUPUEWBODZJGLLRAVCNFSM6AAAAABJSHUN2SVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDCMZZGIYTCOBXGI>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

makeitfuture SRL

PTA. MIHAI VITEAZU, NR.3-4, SUBSOL,
AP.1, MUN. CLUJ-NAPOCA, CLUJ
jud Cluj
Represent Tiberiu Socaci
VAT ID RO45402628
Register No.: J12/6204/2021

***@***.***



This e-mail contains confidential and/or proprietary information. We would like to point out that your data will be processed within the scope of this e-mail; our data protection declaration applies. Your data is necessary for the processing of your request and for the usual business transaction. Further information can be requested at contact at makeitfuture.com. If you are not the correct addressee, or have received this email in error, please notify the sender and delete this email. Unauthorized copying and unauthorized disclosure of this e-mail and the information contained therein is not permitted.

vmanoharas commented:
Hello @anemeswmf, 

Kindly follow the process mentioned below for new/update connector.

Dear Partner, 

Congratulations your PR is approved !! We are proud to announce the Brand-New Certification experience for you to certify Power Platform Copilot Connectors & Plugins. Read the blog here- [Announcing Partner Center to certify and publish Power Platform Copilot connectors and plugins](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpowerautomate.microsoft.com%2Fen-us%2Fblog%2Fannouncing-partner-center-to-certify-and-publish-power-platform-copilot-connectors-and-plugins%2F&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390444301%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=5MYg0GlCsic2JqzwJU0h7mIOc7uHOeL7ZWSbgq%2FFkWk%3D&reserved=0)

As next steps: 

1.	Learn about the new certification experience- [Get your Power Platform connector and plugin certified - Overview](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390456966%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=uWDvs%2BmTbozw1oPc%2FNDNWJ%2FHgnUSfFlRCB1T8UHBh3A%3D&reserved=0)
2.	Package your connector and plugin files- [Prepare Power Platform connector and plugin files for certification](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-submission&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390465418%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=pDuv4bnya1eP3Rrs3p4ArNOXw%2FQipWU87q5oZ0VSIe0%3D&reserved=0)
3.	Initiate a certification request in Partner Center- [Verified publisher certification process](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fsubmit-for-certification&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390471799%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=zbm2px5VZ2ABLBFcORH6tu1fsC9KWdXkP1XJFyluMzw%3D&reserved=0)
4.	Ensure your connector & plugin files comply to the Marketplace policies here- [5000 Power Platform Connector Policies for Marketplace](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Flegal%2Fmarketplace%2Fcertification-policies%235000-power-platform-connectors&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390477736%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=h0pI5J5buo82xxvnBVmrAN8K%2F%2BK8xZlUdcSwzWoqWq0%3D&reserved=0)
5.	Test your connector post certification to provide go-live for deployment- [Testing Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-testing&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390483743%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=Z8pSnhCArFRzEr373o29qM5h5VbV1nlN2gVbv%2Blk1aE%3D&reserved=0)
6.	If you wish to Update your connector or plugin in future- [Updating Guidelines](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fconnectors%2Fcustom-connectors%2Fcertification-updates&data=05%7C02%7Cv-manoharas%40microsoft.com%7C3fecd63b7d2c4d65d68208dc7bdf57ba%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638521446390489681%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=CvxTWJFE3bTFeJ3nDJoCSpzvimhVV9go4cBG%2FQ4gMvo%3D&reserved=0)

For any queries or concerns please get in touch with us on [certificationteam@microsoft.com](mailto:certificationteam@microsoft.com)

Many thanks,
Power Platform 
Copilot connector & plugin certification team
